clients/discord-bot/src/commands/member-commands/help/util.ts

```diff
@@ -5,10 +5,14 @@ import {Unicode} from "../../../enums/unicode";
 import type {CommandSpec} from "../../../marshal";
 
 export const specToField = (spec: CommandSpec): EmbedFieldData => {
-    const aliases: string | undefined = spec.aliases ? spec.aliases.map(a => `\`${config.bot.prefix}${a}\``).join(", ") : undefined;
+    const aliases: string | undefined = spec.aliases
+        ? spec.aliases.map(a => `\`${config.bot.prefix}${a}\``).join(", ")
+        : undefined;
     const docs: string = spec.longDocs ?? spec.docs;
     return {
-        name: `\`${config.bot.prefix}${spec.name}${spec.args.map(a => ` [${a.name}]`).join("")}\``,
+        name: `\`${config.bot.prefix}${spec.name}${spec.args
+            .map(a => ` [${a.name}]`)
+            .join("")}\``,
```

Comment:
This also seems a little odd; breaking up a template string in this way makes it less legible imo.

For what it's worth, it wasn't particularly legible to begin with

Reply:
Fixed here https://github.com/SprocketBot/sprocket/pull/327/commits/973a46d75de2031f225fff5fab8f1953d63f4976

But also agreed, this wasn't legible to begin with 😅